app/models/person_escort_record.rb

```diff
@@ -77,6 +77,17 @@ def update_status!
     save!
   end
 
+  def set_status!(new_status)
```

Comment:
This feels a little strange... it only supports changing status to one value but saves the PER regardless. Maybe renaming the method would make the intent clearer.. or perhaps from the controller call `confirm!` directly.